### PR TITLE
@eessex => [Article] open auth modals from tooltips

### PR DESCRIPTION
**Pull request by l2succes**
Depends on: https://github.com/artsy/reaction/pull/688

### CODE REVIEW

File: src/desktop/apps/article/components/InfiniteScrollArticle.js [hunk 1]
```diff
@@ -135,7 +136,7 @@ export default class InfiniteScrollArticle extends React.Component {
   }
 
   renderContent = () => {
-    const { showTooltips, showToolTipMarketData } = this.props
+    const { showTooltips, showToolTipMarketData, onOpenAuthModal } = this.props
```
Comment (eessex) [inline src/desktop/apps/article/components/InfiniteScrollArticle.js]: so does this mean we will be passing `onOpenAuthModal` down the chain to individual tooltips?

Comment (l2succes) [reply src/desktop/apps/article/components/InfiniteScrollArticle.js]: @eessex Yes, exactly. You can see how that's wired up in this PR https://github.com/artsy/reaction/pull/688

**Comment by eessex** [comment]
This looks ready to go, but want to bump the reaction version to include changes there?

**Comment by eessex** [comment]
oops just realized we are still waiting on a ❌ in the reaction side

**Comment by l2succes** [comment]
Yes, need to bump the reaction version also to include the changes

**Comment by eessex** [comment]
Looks like this just needs to pull a recent master.

**Pull request merged**